sentry-android-core/src/main/java/io/sentry/android/core/AndroidContinuousProfiler.java

```diff
@@ -138,14 +155,29 @@ private synchronized void stop(final boolean restartProfiler) {
       return;
```

Comment:
I'm thinking we should also reset the `chunkId` and the `profilerId` (in case of manual stop) in this case right?

Reply:
yes, we do it [later](https://github.com/getsentry/sentry-java/pull/3711/files#diff-282328031af7b576dd167682cc966fe6852b8b0a6edaf27cc1c68427497d0ad2R180) in this function
But we can add it here, too

Reply:
yeah I guess we have to, because if `endAndCollect` returns null, we're returning early and not resetting the ids, so the next time we start we'll use the wrong ids